src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -141,8 +141,18 @@ def _get_ext_location_definitions(self, missing_locations: list[ExternalLocation
         tf_script = []
         cnt = 1
         for loc in missing_locations:
-            if loc.location.startswith("s3://"):
-                res_name = loc.location[5:].rstrip("/").replace("/", "_")
+            if loc.location.startswith(("s3://", "s3a://", "s3n://")):
```

Comment:
```suggestion
            for prefix in ("s3://", "s3a://", "s3n://"):
                prefix_len = len(prefix)
```
please make it more readable and add a unit test.